app/views/videos/show.html.haml

```diff
@@ -39,6 +39,6 @@
                 %span Rating: #{review.rating} / 5
                 %p 
                   by
-                  = link_to "#{review.user.full_name}"
+                  = link_to "#{review.user.full_name}", user_path(review.user)

```

Comment:
True. I have done this a thousand times but keep making the same error. Sorry. I think i have to concentrate more and stop thinking in the next thing i have to do before ending the current one.